config/software/unixodbc.rb

```diff
@@ -1,8 +1,8 @@
 name "unixodbc"
-default_version "2.3.4"
+default_version "2.3.5"
 
-version "2.3.4" do
-  source :sha256 => "2e1509a96bb18d248bf08ead0d74804957304ff7c6f8b2e5965309c632421e39"
+version "2.3.5" do
+  source :sha256 => "760972e05cc6361aee49d676fb7da8244e0f3a225cd4d3449a951378551b495b"
```

Comment:
Unixodbc is already being updated via https://github.com/DataDog/omnibus-software/pull/345